terraform/state_test.go

```diff
@@ -1062,3 +1121,34 @@ func TestParseResourceStateKey(t *testing.T) {
 		}
 	}
 }
+
+const testV2State = `{
+    "version": 2,
```

Comment:
Is this V2 or V3? Seems like the stored number is 0 based and the comment / function reference is one-based?